CONTRIBUTING.md

```
@@ -78,6 +78,18 @@ For more detail, see the [pull request submission guide](docs/en_US/submitting_p
 
 Contributors are responsible for understanding, testing, and standing behind everything they submit, whether or not AI tools were used. Please do not open large numbers of auto-generated issues or pull requests without first discussing that work with the maintainers.
 
+- If you used AI tools in preparing your PR, please commit with the `Co-Authored-By`, `Generated-by`, or similar commit trailer.
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)